src/contract_wrappers/tokens/security_token_wrapper/common.ts

```diff
@@ -1139,7 +696,7 @@ interface ProduceAddModuleInformation {
 /**
  * This class includes the functionality related to interacting with the SecurityToken contract.
  */
-export default class SecurityTokenWrapper extends ERC20TokenWrapper {
+export default class SecurityTokenCommon extends ERC20TokenWrapper {
```

Comment:
This class should be abstract

Reply:
oh! that make sense and I can remove subscribeAsync and getLogsAsync and also their types